temp_library/helpers.py

```diff
@@ -47,36 +47,39 @@ def statsdsply(trait1, trait2, trait3):
     print()
 
 
-def roll(num, arg):
+def roll(trait, arg):
     # dice rolling function
     input("press enter to roll the dice...")
-    dice = 0
-    for i in range(1):
-        dice += random.randint(1, 6)
+    dice_roll = 0
+    dice_num = trait.getDiceNum()
+    Trait_value = trait.getValue()
+
+    for i in range(dice_num):
+        dice_roll += random.randint(1, 6)
 
     if arg == 'less':
-        if dice <= num:
+        if dice_roll <= Trait_value:
             print("")

```

Comment:
Return a value, don't print (so you could use a different presentation layer (like a website) later on).